tests/lib/rules/no-unused-state.js

```diff
@@ -0,0 +1,438 @@
+/**
+ * @fileoverview Tests for no-unused-state
+ */
+
+'use strict';
+
+const rule = require('../../../lib/rules/no-unused-state');
+const RuleTester = require('eslint').RuleTester;
+
+const parserOptions = {
+  ecmaVersion: 6,
+  ecmaFeatures: {
+    jsx: true
+  }
+};
+
+const eslintTester = new RuleTester({
+  parserOptions: parserOptions,
+  parser: 'babel-eslint'
```

Comment:
is babel-eslint needed for all these test cases? It'd be ideal to use the default parser whenever possible.